content/operate/rs/references/rest-api/objects/bdb/search.md

```diff
@@ -0,0 +1,56 @@
+| search-fork-gc-clean-threshold | integer (range: 1-9223372036854775807) (default: 100) | The fork GC will only start to clean when the number of not cleaned documents exceeds this threshold; otherwise, it will skip this run. |
+| search-vss-max-resize | integer (range: 1-4294967295) (default: 100) | The maximum memory resize for vector similarity indexes in bytes. |
+| search-union-iterator-heap | integer (range: 1-9223372036854775807) (default: 20) | The minimum number of iterators in a union from which the iterator will switch to heap-based implementation. |
+| search-min-phonetic-term-len | integer (range: 1-9223372036854775807) (default: 3) | Minimum length of term to be considered for phonetic matching. |
+| search-multi-text-slop | integer (range: 1-4294967295) (default: 100) | Set RediSearch delta used to increase positional offsets between array slots for multi-text values. Requires a database restart to take effect. |
+| search-raw-docid-encoding | boolean (default: false) | Turn off compression for DocID inverted index. Boost CPU performance. Requires a database restart to take effect. |
+| search-_print-profile-clock | boolean (default: true) | Turn off print of time for ft.profile. For testing only. |
+| search-_free-resource-on-thread | boolean (default: true) | Determine whether some index resources are free on a second thread |
+| search-_numeric-compress | boolean (default: false) | Enable legacy compression of double to float. |
+| search-bg-index-sleep-gap | integer (range: 1-4294967295) (default: 100) | The number of iterations to run while performing background indexing before we call usleep(1) (sleep for 1 micro-second) and make sure that we allow Redis to process other commands. Requires a database restart to take effect. |
+| search-_numeric-ranges-parents | integer (range: 0-2) (default: 0) | Keep numeric ranges in numeric tree parent nodes of leaves for `x` |
+| search-fork-gc-sleep-before-exit | integer (range: 0-9223372036854775807) (default: 0) | Set the number of seconds for the fork GC to sleep before exists, should always be set to 0 (other then on tests). |
+| search-no-mem-pools | boolean (default: false) | Set RediSearch to run without memory pools. Requires a database restart to take effect. |
+| search-_prioritize-intersect-union-children | boolean (default: false) | Intersection iterator orders the children iterators by their relative estimated number of results in ascending order, so that if we see first iterators with a lower count of results, we will skip a larger number of results, which translates into faster iteration. If this flag is set, we use this optimization in a way where union iterators are being factorized by the number of their own children, so that we sort by the number of children times the overall estimated number of results instead. |
```

Comment:
The first sentence is a little hard to follow but I'm not sure of how to make it clearer, since I'm not familiar with this stuff.